src/classes/CW_LocalisedAssetTriggerHelper.cls

```diff
@@ -0,0 +1,78 @@
+/**
+* File Name   :    CW_LocalisedAssetTriggerHelper.cls
+* Description :    B-114115 Denormalised list of labels, products, country, languages
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 06/05/2014 Klea Kolaric	Initial version
+* 0.2 25/06/2014 Cristina P		Changed the class from "with sharing" to "without sharing" so the user can create related records while cost/brief is in approval
+*
+*
+*/
+
+public without sharing class CW_LocalisedAssetTriggerHelper {
+	
+	private Boolean m_isExecuting = false;
+
+    public CW_LocalisedAssetTriggerHelper(Boolean isExecuting) {
+        m_isExecuting = isExecuting;
+    }
+    
+    public void OnAfterInsert( CW_Localised_Asset__c[] newObjects, Map<Id,CW_Localised_Asset__c> newObjectMap ) {
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest rename accordingly.